pages/index.jsx

```diff
@@ -1,4 +1,5 @@
-import { AppLayout, Placeholder, Hero, Board, Form } from '@/components'
+import { AppLayout, Hero, Board, Form } from '@/components'
+import AboutUs from '@/components/AboutUs/AboutUs'
```

Comment:
Please keep all the components imports on the same import

Reply:
When i keep all components in
<img width="1040" alt="Screen Shot 2023-03-22 at 22 32 02" src="https://user-images.githubusercontent.com/110069912/227085240-98ac7d02-3cd1-4991-aa09-7afa3fcfdb86.png">
 the same import I'm having this error

Reply:
<img width="1040" alt="Screen Shot 2023-03-22 at 22 32 02" src="https://user-images.githubusercontent.com/110069912/227085278-d6f0d0c2-b92d-4823-a339-dd3713d39504.png">
When I keep all components on the same import I'm having this error
I tried with banner and contacts components too and I had the same error

Reply:
There's a way to debug it. Check the imports and exports on the components folder

Reply:
![image](https://user-images.githubusercontent.com/110069912/227097796-c34b4edc-2ac7-4a81-89b7-501127c21e5a.png)
I tried to debug it but those components are just rendering when are imported through their subdirectories

Reply:
@marceloglacial do you have a suggestion to solve this issue?

Reply:
Let's keep it as is